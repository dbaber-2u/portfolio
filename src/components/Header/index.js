import React from 'react';
import Nav from '../Nav';

function Header({navigation=[], currentNavigation, setCurrentNavigation}) {
    return (
        <header className="flex-row px-1">
            <h2 className="mx-1">David Baber</h2>
            <Nav navigation={navigation} currentNavigation={currentNavigation} setCurrentNavigation={setCurrentNavigation} />
        </header>
    );
};

export default Header;