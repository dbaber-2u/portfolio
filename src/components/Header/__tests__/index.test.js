import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Header from '..';


const currentNavigation = { "name": "NavItem" };
afterEach(cleanup);



describe('Header component', () => {
    // baseline test
    it('renders', () => {
        render(<Header currentNavigation={currentNavigation} />);
    });

    // snapshot test
    it('matches snapshot', () => {
        const { asFragment } = render(<Header currentNavigation={currentNavigation} />);

        expect(asFragment()).toMatchSnapshot();
    });
})