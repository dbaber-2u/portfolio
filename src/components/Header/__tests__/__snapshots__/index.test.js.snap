// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component matches snapshot 1`] = `
<DocumentFragment>
  <header
    class="flex-row px-1"
  >
    <h2
      class="mx-1"
    >
      David Baber
    </h2>
    <nav>
      <ul
        class="flex-row"
      />
    </nav>
  </header>
</DocumentFragment>
`;
